db/util_testing.go

```diff
@@ -185,3 +183,98 @@ func WaitForUserWaiterChange(userWaiter *ChangeWaiter) bool {
 	}
 	return isChanged
 }
+
+// ViewsAndGSIBucketReadier empties the bucket, initializes Views, and waits until GSI indexes are empty. It is run asynchronously as soon as a test is finished with a bucket.
+var ViewsAndGSIBucketReadier base.TBPBucketReadierFunc = func(ctx context.Context, b *base.CouchbaseBucketGoCB, tbp *base.GocbTestBucketPool) error {
```

Comment:
Why does this do both Views and GSI (when TestsDisableGSI=false), since test runs are going to be doing either one or the other?

Reply:
I might be misremembering, but I thought our tests use a mix of views and GSI. I'll revisit this.

Reply:
As discussed, spun out into CBG-818